website/docs/components/card/index.md

```diff
@@ -7,6 +7,7 @@ links:
   figma: https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?node-id=2359%3A19245&t=XC8SUxxJOFHgqYzK-1
   github: https://github.com/hashicorp/design-system/tree/main/packages/components/addon/components/hds/card
 previewImage: assets/illustrations/components/card.jpg
+keywords: ['tile', 'container']
```

Comment:
```suggestion
keywords: ['tile', 'container', 'box']
```
Might also add `box` here

Reply:
+1 I've called this kind of container a "box" in the past